### PR TITLE
Clean up old `replatforming` test tags.

### PR DESCRIPTION
**Pull request by sengi**
These are no longer relevant since Smokey no longer runs on the old Puppet-based infrastructure.

Tested: ran in staging.

### CODE REVIEW

**Comment by sengi** [comment]
cc @ChrisBAshton

File: README.md [hunk 1]
```diff
@@ -70,28 +69,31 @@ k get secret smokey-signon-account -oyaml | yq .data.email | base64 -d
...
+- govuk_proxy_runner.rb running in a separate process
 
-```shell
-shell1$ env GOVUK_PROXY_PROFILE=mirrorS3 ruby govuk_proxy_runner.rb
+To run Smokey against the GOV.UK S3 mirror:
 
-shell2$ env GOVUK_PROXY_PROFILE=mirrorS3 ENVIRONMENT=production bundle exec cucumber --tags="@worksonmirror"
+```sh
+export GOVUK_PROXY_PROFILE=mirrorS3
+ruby govuk_proxy_runner.rb
```
Comment (ChrisBAshton) [inline README.md:81]:
```suggestion
ruby govuk_proxy_runner.rb &
```

**Review by ChrisBAshton — APPROVED** [review]
⭐

**Review by theseanything — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sengi) [reply README.md]: Oh good catch, thanks!

**Pull request merged**